### PR TITLE
Added custom.less file.

### PR DESCRIPTION
**Pull request by justincron**

### CODE REVIEW

File: js/tests/unit/tooltip.js [hunk 1]
```diff
@@ -680,7 +680,47 @@ $(function () {
     $styles.remove()
   })
 
+<<<<<<< HEAD
```
Comment (kkirsche) [inline js/tests/unit/tooltip.js]: Leftover merge error it seems

File: js/tests/unit/tooltip.js [hunk 2]
```diff
@@ -680,7 +680,47 @@ $(function () {
...
+
+    $target.bootstrapTooltip('hide')
+
+    $container.remove()
+    $styles.remove()
+    window.scrollTo(0, 0)
+  })
+
+  test('should not error when trying to show an auto-placed tooltip that has been removed from the dom', function () {
+>>>>>>> FETCH_HEAD
```
Comment (kkirsche) [inline js/tests/unit/tooltip.js]: Leftover merge error it seems

**Comment by twbs-savage** [comment]
**Tests failed.** Automated cross-browser testing via Sauce Labs and Travis CI shows that the JavaScript changes in this pull request are: **BUSTED**

Commit: 937854b082f8b8d5f3b78515b629ad36272f02d9
Build details: https://travis-ci.org/twbs-savage/bootstrap/builds/52829366

(*Please note that this is a [fully automated](https://github.com/twbs/savage) comment.*)

**Comment by cvrebert** [comment]
I don't quite see the value here. You might as well just `@import "bootstrap.less"` from your own main less file instead.

**Comment by justincron** [comment]
Thanks. I just thought it would be nice to already have it there in case you wanted to write your own less/css to overwrite current styles, without having to link to another custom.css file.

**Comment by kkirsche** [comment]
@justincron My concern is for users who use Bootstrap through Bower that they'll be getting extra files for no reason. I appreciate the thought though :smiley:

**Comment by justincron** [comment]
Oh, I see. I understand your concern. Thanks!

**Pull request closed without merging**